### PR TITLE
CLOUDSTACK-8632: Unable to revert snapshot on XenServer

### PR DESCRIPTION
**Pull request by pritisarap12**
On XenServer check that volume snapshot fails for the ROOT volume of VM which has VM snapshot created.

### CODE REVIEW

**Comment by wilderrodrigues** [comment]
Hi @pritisarap12 

The issue/PR title says "Unable to rever snapshot on XenServer", however you have created a component test to "check that volume snapshot fails for the ROOT volume of VM which has VM snapshot created."

Could you please just elaborate a bit more on the meaning of this PR?

Cheers,
Wilder

File: test/integration/testpaths/testpath_revert_snap.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        volumes_cluster_list = list_volumes(
+                self.apiclient,
+                virtualmachineid=vm.id,
+                type='ROOT',
+                listall=True
+                )
+
+        volume_list_validation = validateList(volumes_cluster_list)
+
+        self.assertNotEqual(
```
Comment (sanju1010) [inline test/integration/testpaths/testpath_revert_snap.py]: Instead of this, compare whether the list is returning PASS or not.

Comment (pritisarap12) [inline test/integration/testpaths/testpath_revert_snap.py]: Updated testcase with review changes.

**Comment by wilderrodrigues** [comment]
LGTM

**Pull request closed without merging**